packages/koa-metrics/src/tags.ts

```diff
@@ -0,0 +1,22 @@
+import {Context} from 'koa';
+
+export enum Tags {
+  Path = 'path',
+  RequestMethod = 'request_method',
+  ResponseCode = 'response_code',
+  ResponseType = 'response_type',
```

Comment:
shouldn't this enum include the queue time and such?

Reply:
Those are metrics, whereas these are tags. The idea is that each metric is allowed to have tags associated with it.